src/PrivateDns/PrivateDns/VirtualNetworkLinks/SetAzurePrivateDnsVirtualNetworkLink.cs

```diff
@@ -22,8 +22,8 @@ namespace Microsoft.Azure.Commands.PrivateDns.VirtualNetworkLinks
     /// <summary>
     /// Updates an existing zone.
     /// </summary>
-    [Cmdlet("Update", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "PrivateDnsVirtualNetworkLink", SupportsShouldProcess = true, DefaultParameterSetName = FieldsParameterSetName), OutputType(typeof(PSPrivateDnsVirtualNetworkLink))]
-    public class UpdateAzurePrivateDnsVirtualNetworkLink : PrivateDnsBaseCmdlet
+    [Cmdlet("Set", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "PrivateDnsVirtualNetworkLink", SupportsShouldProcess = true, DefaultParameterSetName = FieldsParameterSetName), OutputType(typeof(PSPrivateDnsVirtualNetworkLink))]
```

Comment:
Changing it to Set for now to keep it uniform across all resources ( rest of them have Set and not Update)